test/functional/mempool_sigoplimit.py

```diff
@@ -133,6 +136,48 @@ def test_sigops_limit(self, bytes_per_sigop, num_sigops):
         assert_equal(entry_parent['descendantcount'], 2)
         assert_equal(entry_parent['descendantsize'], parent_tx.get_vsize() + sigop_equivalent_vsize)
 
+    def test_sigops_package(self):
+        # Make a 2-transaction package which fails vbyte checks even though
+        # separately they would work.
+        self.restart_node(0, extra_args=["-bytespersigop=5000"] + self.extra_args[0])
+
+        _, pubkey = generate_keypair()
+        amount_for_bare = 50000
+        tx_parent_dict = self.wallet.create_self_transfer(fee=Decimal("3"))
+        tx_parent_utxo = tx_parent_dict["new_utxo"]
+        tx_parent = tx_parent_dict["tx"]
+        tx_parent.vout.append(CTxOut(amount_for_bare, keys_to_multisig_script([pubkey], k=1)))
+        tx_parent.vout[0].nValue -= amount_for_bare
+        tx_parent.calc_sha256()
+        tx_parent_utxo["txid"] = tx_parent.hash
+        tx_parent_utxo["value"] -= Decimal("0.00005000")
+
+        tx_child_dict = self.wallet.create_self_transfer(utxo_to_spend=tx_parent_utxo, fee=Decimal("3"))
+        tx_child_utxo = tx_child_dict["new_utxo"]
+        tx_child = tx_child_dict["tx"]
+        tx_child.vout.append(CTxOut(amount_for_bare, keys_to_multisig_script([pubkey], k=1)))
+        tx_child.vout[0].nValue -= amount_for_bare
+        tx_child.calc_sha256()
+        tx_child_utxo["txid"] = tx_child.hash
+        tx_child_utxo["value"] -= Decimal("0.00005000")
+
+        # Separately, the parent tx is ok
+        parent_individual_testres = self.nodes[0].testmempoolaccept([tx_parent.serialize().hex()])[0]
+        assert parent_individual_testres["allowed"]
+        # Multisig is counted as MAX_PUBKEYS_PER_MULTISIG = 20 sigops
+        assert_equal(parent_individual_testres["vsize"], 5000 * 20)
+
+        # But together, it's exceeding limits
+        packet_test = self.nodes[0].testmempoolaccept([tx_parent.serialize().hex(), tx_child.serialize().hex()])
+        assert_equal([x["package-error"] for x in packet_test], ["package-mempool-limits", "package-mempool-limits"])
+
+        # When we actually try to submit, the parent makes it into the mempool, but the child would exceed ancestor vsize limits
+        assert_raises_rpc_error(-26, "too-long-mempool-chain", self.nodes[0].submitpackage, [tx_parent.serialize().hex(), tx_child.serialize().hex()])
```

Comment:
In 97d53ff4c3ede1d1fdc6d93386f14c230b03b98d "Add functional test to catch too large vsize packages"

Can you add a comment explaining what this test is actually testing for? It's not clear to me why `testmempoolaccept` would behave differently from `submitpackage`, and how observing this difference in behavior is useful in a test.